app/services/metrc_service/package/harvest.rb

```diff
@@ -4,7 +4,8 @@ module MetrcService
   module Package
     class Harvest < MetrcService::Package::Base
       def call
-        call_metrc(:create_package, payload, testing?)
+        # finishing packages not yet supported
```

Comment:
Can we raise a "Not supported method" exception on this case, please?

Reply:
Come to think of it, this file is redundant now. If I remove it then we'll get a raise from [here](https://github.com/artemis-ag/apps-cannapis/blob/df9f74c0e2aa17d5b04db9823bf64949a5a35924/app/services/metrc_service.rb#L52) anyway, handled and logged further up the stack.